packages/core/src/modules/connections/ConnectionsApi.ts

```diff
@@ -227,6 +230,42 @@ export class ConnectionsApi {
     return connectionRecord
   }
 
+
+  /**
+   * Send a trust ping to an established connection
+   *
+   * @param connectionId the id of the connection for which to accept the response
+   * @param responseRequested do we want a response to our ping
+   * @param withReturnRouting do we want a response at the time of posting
+   * @returns boolean indicating success or failure
+   */
+  public async trustPing(connectionId: string, responseRequested=true, withReturnRouting=false): Promise<boolean> {
```

Comment:
The DIDComm V2 PR also adds support for the trust ping method. Can we make sure to align this implementation with that method? The method name is `sendPing`.

Reply:
I adjusted to model off of sendPing, so I think it lines up better with sendPing.